BookingsApi/BookingsApi.Infrastructure.Services/ParticipantDtoExtension.cs

```diff
@@ -34,5 +34,19 @@ public static string ExtractJudgeEmail(string[] properties)
                 ? properties[Array.IndexOf(properties, "JudgeEmail") + 1]
                 : string.Empty;
         }
+
+        public static string GetContactEmail(this ParticipantDto participant)
```

Comment:
Do these methods not just do the same thing as the null-coalescing operator: ??

Reply:
I used to string.isNullOrEmpty just to protect against empty strings, as the null coalescing operator won't catch them.

I added these methods just so we have a single defined place for getting the emails and phone numbers, rather than having to check the 2 properties in multiple places